### PR TITLE
fix feedback to really load the correct messages

### PR DESCRIPTION
**Pull request by lckr**
for @marcelhollerbach

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Utils.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+     */
+    public static String loadFile(String file) {
+        try {
+            return CharStreams
+                    .toString(new InputStreamReader(
+                            Main.class.getResourceAsStream(file)
+                            , Charsets.UTF_8));
+        } catch (IOException e) {
+            e.printStackTrace();
+            return "";
```
Comment (TheRittler) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Utils.java]: Is this really wanted? I think it would be better if you throw an exception, if the patch is invalid, the file corrupt etc.

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Utils.java]: I sound log the exception into the logger (2 sec.). And then return ""; the rest of the programm does not have any chance to repaire this...

File: src/test/resources/feedback/workermessage1.txt [hunk 2]
```diff
@@ -10,15 +10,12 @@ Rating:
...
 Rating: 1
-
 Rating:
 
 Feedback: Foobar2
 Quality: 2
 Rating: 2
-
 Additional feedback: awes0me Message
-
```
Comment (TheRittler) [inline src/test/resources/feedback/workermessage1.txt:21]: Here should be a new line for better layout
[This](http://i.imgur.com/3du4fhN.png) looks a bit strange, I think.

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Utils.java (hunk 1 above)
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Utils.java]: Done.

File: src/test/resources/feedback/workermessage1.txt (hunk 2 above)
Comment (marcelhollerbach) [inline src/test/resources/feedback/workermessage1.txt:21]: Okay, I just fixed the test so it works. The problem is somewhere else. This is just a fix that the code is running.

Comment (TheRittler) [inline src/test/resources/feedback/workermessage1.txt:21]: I'd say, functionality should be same or better after changes :stuck_out_tongue_winking_eye: Especially when i don't see, why this change was needed :wink:

Comment (marcelhollerbach) [inline src/test/resources/feedback/workermessage1.txt:21]: the end of the file was here a "\n" which is not generated by the message. Just run the test and see what is happening. If you want to change the layout, change it, this pr is about fixing the bug that the message cannot be loaded since the loader uses relative paths based on the $PWD before, which does not work in any way on a productive system, since $PWD could be anything. While doing this change this test was fixed. (The mocks of the database are illegal the records will be null => NPE for a null record).
